apps/gameserver/include/Character.h

```diff
@@ -24,6 +24,10 @@ class Character : public bango::space::quad_entity
     int             m_z;
     std::uint8_t    m_map;
 
+    std::uint16_t   m_n64GState;
+    std::uint16_t   m_n64MState;
+    std::uint16_t   m_n64GStateEx;
+    std::uint16_t   m_n64MStateEx;
```

Comment:
Those are ROAM conventions (adding `n` before names and UpperCase names)
Stick to the same naming convention as above (m_z/m_map)

Also, the data types seem wrong - only 2 bytes? If I remember correctly those should be 8 bytes and signed (std::int64_t)

```suggestion
    std::int64_t   m_gstate;
    std::int64_t   m_mstate;
    std::int64_t   m_gstate_ex;
    std::int64_t   m_mstate_ex;
```

Reply:
done

Reply:
done